manifests/init.pp

```diff
@@ -57,6 +57,7 @@
   $pip_ensure         = 'installed',
   $pip_package        = $python::params::pip,
   $setuptools_ensure  = 'installed',
+  ${easy_install_pip} = 'pip',
```

Comment:
```suggestion
  $easy_install_pip = 'pip',
```

When you declare a pip variable or use it outside of a string you don't use the `{}`. Braces are for string interpolation.